example/spec/parallel/cnmWorkflow/CnmWorkflowFromSqsSpec.js

```diff
@@ -72,7 +73,9 @@ async function cleanUp() {
   const rules = await readJsonFilesFromDir(ruleDirectory);
   await deleteRules(config.stackName, config.bucket, rules, ruleSuffix);
   await deleteExecution({ prefix: config.stackName, executionArn: workflowExecution.executionArn });
-  await removePublishedGranule({ prefix: config.stackName, granuleId });
+  await removePublishedGranule({ prefix: config.stackName,
+    granuleId,
+    collectionId: constructCollectionId(ruleOverride.collection.name, encodeURIComponent(ruleOverride.collection.version)) });
```

Comment:
Is there some way we could abstract this to a helper function? Perhaps just for tests but it would be nice anywhere we need to make the collectionId. Mainly I'm concerned about remembering to URI encode the version every time.

Reply:
I wonder if it could be done in `constructCollectionId` or if that breaks everything.

Reply:
encoding in the constructCollectionId function